src/libmongoc/doc/installing.rst

```diff
@@ -211,24 +233,7 @@ The ``cmake`` utility is also required. First `install Homebrew according to its
 
   $ brew install cmake
 
-Download the latest release tarball:
-
-.. parsed-literal::
-
-  $ curl -LO https://github.com/mongodb/mongo-c-driver/releases/download/x.y.z/mongo-c-driver-x.y.z.tar.gz
-  $ tar xzf mongo-c-driver-x.y.z.tar.gz
-  $ cd mongo-c-driver-x.y.z
-
-Build and install the driver:
-
-.. code-block:: none
-
-  $ mkdir cmake-build
-  $ cd cmake-build
-  $ cmake -DENABLE_AUTOMATIC_INIT_AND_CLEANUP=OFF ..
-  $ make install
-
-All of the same variations described above (e.g., building only libbson, building documentation, etc.) are available when building on macOS.
+Follow the build instructions above.
```

Comment:
Above is a bit vague. I suggest linking directly to the "Build from release tarball section", which I believe you can do with a sphinx label and reference

`.. build-on-windows` below is one example, and is linked to like:
```
:ref:`when you ran CMake <build-on-windows>`
```

Reply:
Ack.